### ISSUE TITLE
Deis does not support alphanumeric process type in Procfile

### ISSUE THREAD

**Issue by n0n0x** [deis/workflow-cli#88]
Docs says that process types declared in `Procfile` are alphanumeric and can be named arbitrarily: http://docs.deis.io/en/latest/using_deis/process-types/#declaring-process-types

But this is not the case, as when using an actual alphanumeric string as process type, I get the following message:
```
'worker2=1' does not match the pattern 'type=num', ex: web=2
```

github repo to test this out: https://github.com/n0n0x/deis-cli-bug

```
git clone https://github.com/n0n0x/deis-cli-bug.git
cd deis-cli-bug
deis create cli-bug
git push deis master
deis ps:scale worker1=1
deis ps:scale worker2=1
deis ps:scale worker=1
```

The only one that gets scaled is `worker`.

Example output:
```
n0n0x@n0n0x:~/repos/cli-bug (master)$ deis ps:scale worker1=1
'worker1=1' does not match the pattern 'type=num', ex: web=2
Scaling processes... but first, coffee!
done in 0s
=== cli-bug Processes

n0n0x@n0n0x:~/repos/cli-bug (master)$ deis ps:scale worker2=1
'worker2=1' does not match the pattern 'type=num', ex: web=2
Scaling processes... but first, coffee!
done in 0s
=== cli-bug Processes

n0n0x@n0n0x:~/repos/cli-bug (master)$ deis ps:scale worker=1
Scaling processes... but first, coffee!
done in 37s
=== cli-bug Processes
--- worker:
cli-bug-v2-worker-eu6wa up (v2)

**Comment by Joshua-Anderson** [comment]
Sorry, this is a case of a [too strict regexp](https://github.com/deis/workflow-cli/blob/master/cmd/ps.go#L44). I'll fix this up right now!

**Comment by helgi** [comment]
https://github.com/deis/controller/blob/master/rootfs/api/serializers.py#L15-L17 is the API regex for the same thing

**Comment by Joshua-Anderson** [comment]
Fixed in PR #89
